app/views/layouts/main.html.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
Are we going to use HAML or erb?

Reply:
I would stick to ERB, we will not use that much of it anyhow